src/main/java/seedu/address/ui/BrowserPanel.java

```diff
@@ -47,8 +47,7 @@ private void loadPersonPage(Person person) {
         loadPage(SEARCH_PAGE_URL + person.getName().fullName);
     }
 
-    private void loadTaskPage(Task task)
-    {
+    private void loadTaskPage(Task task) {
```

Comment:
Can delete this file

Reply:
or just remove this changes so that there's no merge conflict. My merged code has already removed this file